src/zookeeper_config.py

```diff
@@ -91,4 +107,19 @@ def create_properties(self, config):
             if config.get(k, None) is not None:
                 props += f"\n{v}={config[k]}"
 
+        if self.ssl_enabled():
+            props += TLS_ZOOKEEPER_PROPERTIES
+            props += "\nzookeeper.connect="
```

Comment:
Why does ZooKeeper need to set `zookeeper.connect` property to `zookeeper.properties`? I thought that was a Kafka config option.

Reply:
You are correct.  I misread the documentation there.